frontend/packages/core/src/stories/button-group.stories.tsx

```diff
@@ -25,13 +28,13 @@ Default.args = {
   ],
 };
 
-export const Destructive = Template.bind({});
-Destructive.args = {
```

Comment:
(couldn't comment on lines where changes weren't made) for "Default" - should we update it to "Primary", like in the button story?